app/views/estimate_flow/forms/_other_income.html.slim

```diff
@@ -26,7 +26,7 @@
     h2.govuk-heading-m class="govuk-!-margin-top-9 govuk-!-margin-bottom-2"
       = t("estimate_flow.#{i18n_key}.#{type}.label")
     span.govuk-caption-m class="govuk-!-margin-bottom-4"
-      - if FeatureFlags.enabled?(:disregard_cost_of_living) && type == :other
+      - if type == :other
```

Comment:
If you rename the `estimate_flow.assets(/partner_assets).other.hint_with_col` to `estimate_flow.assets(/partner_assets).other.hint` then we can remove this if/else block entirely.

Reply:
Thanks - I've just worked out that hint_with_col means 'cost_of_living'